modules/test/playwright/playwright.config.ts

```diff
@@ -28,7 +28,7 @@ import {config as featureFlagWebConfig} from './tests/feature-flag-web/config';
 import {config as fragmentWebConfig} from './tests/fragment-web/config';
 import {config as frontendDataSetAdminWebConfig} from './tests/frontend-data-set-admin-web/config';
 import {config as frontendDataSetWebConfig} from './tests/frontend-data-set-web/config';
-import {config as frontendEditorCKEditorSampleWebConfig} from './tests/frontend-editor-ckeditor-sample-web/config';
+import {config as frontendCKEditorSampleWebConfig} from './tests/frontend-ckeditor-sample-web/config';
```

Comment:
Why this change of project name? PW project name should normally match module name, [frontend-editor-ckeditor-sample-web](https://github.com/liferay/liferay-portal/tree/master/modules/apps/frontend-editor/frontend-editor-ckeditor-sample-web). There is an argument to be had if it should be `frontend-editor-ckeditor-web` (no `sample`), but `frontend-ckeditor-sample-web` seems just wrong.

Reply:
Hello @markocikos 
Having the same name, caused issues for CI as it had could not find frontend-editor-ckeditor-sample-web. Changing the name, made the module deploy correctly and pass the PW tests.

Here is a link about the situation.
https://liferay.slack.com/archives/C19PWQ74J/p1727287797591529

Let me know if you want things to be changed.

Reply:
@fortunatomaldonado If I understand the problem correctly, this definitely not the proper solution. If a module cannot be found, how can renaming to a non-existing module possibly work? Something very hacky is going on here. The solution is to fix module deployment in `frontend-editor-ckeditor-sample-web`, to use the correct configuration format, and add the new test to it.

Reply:
I pulled your PR and and the tests are locally not even showing up in framework. That's why CI is green, tests are not run at all.

![image](https://github.com/user-attachments/assets/b76d08da-7c2d-4856-b325-4eb746c6789f)

Reply:
Origin of this change is in this [slack conversation](https://liferay.slack.com/archives/C19PWQ74J/p1727287797591529). Let's keep talking there, this looks a bit odd to me as well

Reply:
probably, removing the `-sample` part in the PW project folder will work 
That's how we arranged tests in `frontend-data-set-web` pw folder. There we deploy `frontend-data-set-sample-web` 

So, pw folder here should be `frontend-editor-ckeditor-web` and there we'll deploy the sample-web module